<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>298</y>
     <width>467</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QPushButton" name="resetAllButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reset All Settings</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>111</y>
     <width>470</width>
     <height>69</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="sourceBox1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>3</y>
      <width>165</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>7</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I/Q Input</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialBox1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>4</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="iqEnableBalance">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>24</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enable I/Q Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="iqBalancePhaseLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>44</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Phase: 0.123</string>
    </property>
   </widget>
   <widget class="QSlider" name="iqBalanceGain">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>26</y>
      <width>155</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>-500</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="iqBalancePhase">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>27</y>
      <width>155</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-500</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="iqBalanceGainLabel">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>47</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gain: 0.123</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iqBalanceReset">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>43</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sdrOptionsButton">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>3</y>
      <width>157</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SDR Options (must be on)</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>16</y>
     <width>470</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="sdrEnabledButton1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>19</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="sdrEnabledButton2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>24</y>
      <width>19</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="sdrEnabledButton3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>44</y>
      <width>19</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="sdrEnabledButton4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>64</y>
      <width>19</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverBox1">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>5</y>
      <width>142</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverBox2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>26</y>
      <width>143</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverBox3">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>46</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverBox4">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>66</y>
      <width>140</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>1</y>
      <width>281</width>
      <height>89</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="outputBox1">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>4</y>
       <width>165</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>5</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Audio Output</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>49</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IQ Order</string>
     </property>
    </widget>
    <widget class="QComboBox" name="IQSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>47</y>
       <width>114</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="startupBox1">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>67</y>
       <width>94</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="startupEdit1">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>68</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Startup</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>49</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>I/Q Gain</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="iqGain1">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>47</y>
       <width>49</width>
       <height>18</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>25</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SampleRate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sampleRateBox1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>25</y>
       <width>90</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <zorder>outputBox1</zorder>
    <zorder>label_9</zorder>
    <zorder>label_11</zorder>
    <zorder>IQSettings</zorder>
    <zorder>startupBox1</zorder>
    <zorder>startupEdit1</zorder>
    <zorder>label_5</zorder>
    <zorder>label_2</zorder>
    <zorder>iqGain1</zorder>
    <zorder>frame_2</zorder>
    <zorder>label_10</zorder>
    <zorder>sampleRateBox1</zorder>
   </widget>
   <zorder>sdrEnabledButton1</zorder>
   <zorder>sdrEnabledButton2</zorder>
   <zorder>sdrEnabledButton3</zorder>
   <zorder>sdrEnabledButton4</zorder>
   <zorder>receiverBox1</zorder>
   <zorder>receiverBox2</zorder>
   <zorder>receiverBox3</zorder>
   <zorder>receiverBox4</zorder>
   <zorder>sourceBox1</zorder>
   <zorder>label_8</zorder>
   <zorder>frame_3</zorder>
  </widget>
  <widget class="QWidget" name="sdrSpecific" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>187</y>
     <width>473</width>
     <height>108</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
